src/set.rs

```diff
@@ -249,6 +257,39 @@ mod tests {
     const TEST_BATCH_SMALL: usize = 100;
     const TEST_REP: usize = 20;
 
+    #[test]
+    fn debug_when_empty() {
+        let set: ConSet<String> = ConSet::new();
+        assert_eq!("{}".to_owned(), format!("{:?}", set));
```

Comment:
Nitpick: shorter way would be to compare the references (and wouldn't require another allocation), eg:

```rust
assert_eq!("{}", &format!(…));
```

Reply:
Nice.